etc/profile-m-z/rtv-addons.inc

```diff
@@ -0,0 +1,23 @@
+# This file is overwritten during software install.
+# Persistent customizations should go in a .local file.
+
+# You can configure rtv to open different type of links
```

Comment:
perhaps add a note too rtv.profile so users are made aware of this extra configuration

Reply:
Suggestion: add something like the below to rtv.profile:
```
$ cat /etc/firejail/rtv.profile
...
# Allow python (blacklisted by disable-interpreters.inc)
include allow-python2.inc
include allow-python3.inc

# Uncomment or put in rtv.local for external application support
#include rtv-addons.inc

include disable-common.inc
```

Reply:
ok